src/components/CentralContainer/Sidebar/SidebarTabs/SearchTabPanel/Result.tsx

```diff
@@ -36,8 +37,10 @@ const Result = ({logEventNum, message, matchRange}: ResultProps) => {
         message.slice(...matchRange),
         message.slice(matchRange[1]),
     ];
+    const setLogEventNum = useViewStore((state) => state.setLogEventNum);
     const handleResultButtonClick = () => {
         updateWindowUrlHashParams({logEventNum});
+        setLogEventNum(logEventNum);
     };
```

Comment:
```suggestion
    const handleResultButtonClick = useCallback(() => {
        updateWindowUrlHashParams({logEventNum});
        setLogEventNum(logEventNum);
    }, [logEventNum]);
```
This is my fault. We should use `useCallback()` hook for this function to avoid repetitive function rendering. I'm not adding `setLogEventNum` to the dependency array, since my previous comment suggests we should use `getState()` to get `setLogEventNum` inside this function.